tutorials/08-tdem/plot_inv_1_em1dtm.py

```diff
@@ -260,7 +260,7 @@
 
 # Directives for the IRLS
 update_IRLS = directives.Update_IRLS(
-    max_irls_iterations=30, minGNiter=1, coolEpsFact=1.5, update_beta=True
+    max_irls_iterations=30, minGNiter=1, coolEpsFact=1.5
 )
```

Comment:
Same for this example

Reply:
Had to do this in the previous implementation as the old class was getting overloaded with the new old. Reverting.